src/App.js

```diff
@@ -1,17 +1,30 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './App.css';
-
+import todosFromServer from './api/todos';
 import users from './api/users';
+import { TodoList } from './components/TodoList/TodoList';
+import { Form } from './components/Form/Form';
 
 function App() {
+  const [todos, addTodos] = useState(todosFromServer);
```

Comment:
```suggestion
  const [todos, setTodos] = useState(todosFromServer);
```
зазвичай цю функцію називають саме так щоб виділити її серед інших